config/config.go

```diff
@@ -33,28 +34,30 @@ func (c *Config) GetCurrencyConfig() CurrencyConfig {
 
 // GetExchangeBankAccounts returns banking details associated with an exchange
 // for depositing funds
-func (c *Config) GetExchangeBankAccounts(exchangeName, depositingCurrency string) (BankAccount, error) {
+func (c *Config) GetExchangeBankAccounts(exchangeName, id, depositingCurrency string) (*banking.Account, error) {
 	m.Lock()
 	defer m.Unlock()
 
 	for x := range c.Exchanges {
 		if strings.EqualFold(c.Exchanges[x].Name, exchangeName) {
 			for y := range c.Exchanges[x].BankAccounts {
-				if strings.Contains(c.Exchanges[x].BankAccounts[y].SupportedCurrencies,
-					depositingCurrency) {
-					return c.Exchanges[x].BankAccounts[y], nil
+				if strings.EqualFold(c.Exchanges[x].BankAccounts[y].ID, id) {
+					if strings.Contains(c.Exchanges[x].BankAccounts[y].SupportedCurrencies,
```

Comment:
I think this should be changed to `if common.StringDataCompareInsensitive(strings.Split(c.Exchanges[x].BankAccounts[y].SupportedCurrencies, ",")` to allow for casing. I submitted `usd` as my currency, got rejected, then did `USD` and was fine.
Also, its case-intensive for crypto withdrawals

Reply:
Yeah I will make the change now there are a few other places that could do with case intensive checking